src/api/handlers/PetUpdateHandler.ts

```diff
@@ -0,0 +1,29 @@
+import { Handler } from "aws-lambda";
+import { PetService } from "../services/PetService";
+
+class PetUpdateHandler {
+    private service: PetService;
+
+    constructor() {
+        this.service = new PetService();
+    }
+
+    public execute(id:number, event: any) {
```

Comment:
- @Aristevao please update this part:
```typescript
class PetUpdateHandler {
    private service: PetService;

    constructor() {
        this.service = new PetService();
    }

    public execute(event: any) {
        const pet = this.service.update(event.pathParameters.id, event.body); // here we get id and body
        const response = {
            statusCode: 200,
            headers: {
                "x-custom-header": "My Header Value",
                "Access-Control-Allow-Origin": "*", // Required for CORS support to work
                "Access-Control-Allow-Credentials": true, // Required for cookies, authorization headers with HTTPS
            },
            body: JSON.stringify({ pet }),
        };
        return response;
    }
}

export const handler: Handler = async (event, context, callback) => {
    const response = new PetUpdateHandler().execute(event); // here I just pass on event object
    callback(null, response);
};
```
- Please, write manually the code, and pay attention to the differences